test/Garnet.test/RespListTests.cs

```diff
@@ -1591,5 +1591,49 @@ public void LPOSWithInvalidOptions(string items, string find, string expectedInd
         }
 
         #endregion
+
+        #region ZREVRANGEBYLEX
+
+        [Test]
+        [TestCase("(a", "(a", new string[] { })]
```

Comment:
Perhaps add some more test cases?

Reply:
I have mentioned the reason in the description
> Note: ZRANGE with REV command has a lot of issues. Also, all the causes are failing. I will have a separate PR to fix it. This is the reason I couldn't write test causes for ZREVRANGEBYLEX command with any of the valid cases.